src/sonic-yang-models/yang-templates/sonic-acl.yang.j2

```diff
@@ -144,7 +152,7 @@ module sonic-acl {
 
 				leaf ETHER_TYPE {
 					type string {
-						pattern "(0x88CC|0x8100|0x8915|0x0806|0x0800|0x86DD|0x8847)";
+					    pattern "0x[6-9a-fA-F][0-9a-fA-F]{2}|0x[1-9a-fA-F][0-9a-fA-F]{3}|0x0806|0x0800";
```

Comment:
0x0[6-9a-fA-F][0-9a-fA-F]{2}

Reply:
Leading zeros as no significance and this is represented as hex number. Requester has to take the extra step to add a 0 which is not useful